Source/cmake/OptionsHaiku.cmake

```diff
@@ -213,12 +214,13 @@ if (CMAKE_COMPILER_IS_GNUCC AND "${LOWERCASE_CMAKE_HOST_SYSTEM_PROCESSOR}" STREQ
 	string(APPEND CMAKE_CXX_FLAGS " -march=pentium4")
 endif()
 
-SET_AND_EXPOSE_TO_BUILD(USE_TEXTURE_MAPPER OFF)
+SET_AND_EXPOSE_TO_BUILD(USE_COORDINATED_GRAPHICS ON)
+SET_AND_EXPOSE_TO_BUILD(USE_TEXTURE_MAPPER ON)
+SET_AND_EXPOSE_TO_BUILD(USE_NICOSIA ON)
 SET_AND_EXPOSE_TO_BUILD(USE_GRAPHICS_LAYER_TEXTURE_MAPPER OFF)
 SET_AND_EXPOSE_TO_BUILD(ENABLE_GRAPHICS_CONTEXT_GL OFF)
-SET_AND_EXPOSE_TO_BUILD(USE_EGL FALSE)
-SET_AND_EXPOSE_TO_BUILD(USE_TILED_BACKING_STORE OFF)
-SET_AND_EXPOSE_TO_BUILD(USE_COORDINATED_GRAPHICS OFF)
+SET_AND_EXPOSE_TO_BUILD(USE_EGL FALSE) # not used anywhere except in Haiku-specific code. Remove?
+SET_AND_EXPOSE_TO_BUILD(USE_TILED_BACKING_STORE OFF) # not used anywhere except in Haiku-specific code. Remove?
```

Comment:
yes, it looks like these are old options that don't exist anymore, so this can be removed, and the code still depending on them (and so, always disabled) can also be removed.